### ISSUE TITLE
Exclude Overarching categories from Random Sort 

### ISSUE THREAD

**Issue by halfempty** [everythingtype/calcott#44]
@ncalcott I have this note:

> Remove overarching categories showing up in Random Sort like http://www.nicholascalcott.com/index/miscellaneous/

It looks like you've done this by adding term exclusions. Is this working well? If yes, I won't spend any time on it. Or, if it's not working, I can look at automating it. Let me know!

**Comment by ncalcott** [comment]
No, the problem is that it doesn't seem to be working, despite creating the exclusions...  I couldn't figure out why, but I'd also prefer not to have to manually enter the exclusions each time I create a new tag category (though this admittedly isn't often).
